src/components/Synthetics/GmSwap/GmSwapBox/useUpdateByQueryParams.tsx

```diff
@@ -68,6 +74,19 @@ export function useUpdateByQueryParams({
         }
       }
 
+      if (pickBestGlv) {
+        setOperation(Operation.Deposit);
+
+        const glvs = Object.values(marketsInfo).filter((m) => isGlv(m)) as GlvInfo[];
+
+        if (glvs.length) {
+          const bestGlv = glvs.reduce((best, glv) => {
+            return (best.glvToken.totalSupply ?? 0n) > (glv.glvToken.totalSupply ?? 0n) ? best : glv;
+          });
+          marketAddress = bestGlv.indexTokenAddress.toLowerCase();
```

Comment:
I don't know this place but marketAddress always refers to market, not a token. But here it's a token address. This is incorrect or misleading

Reply:
This hook updates Market selector in Buy/Sell card. I don't think that adding extra query parameter just to note that it's glv and then eventually change same UI part is good idea. I guess if as I removed `indexTokenAddress` and here now is `marketTokenAddress` it will be more clear